mPowerSDK/TasksAndSteps/SpatialSpanMemoryControllers/APHSpatialSpanMemoryGameViewController.m

```diff
@@ -72,24 +63,6 @@ @implementation APHSpatialSpanMemoryGameViewController
 
 #pragma  mark  -  Task View Controller Delegate Methods
 
-- (void)taskViewController:(ORKTaskViewController *) __unused taskViewController stepViewControllerWillAppear:(ORKStepViewController *)stepViewController
-{
-    
-    if ([stepViewController.step.identifier isEqualToString:kConclusionStepIdentifier]) {
-        [[UIView appearance] setTintColor:[UIColor appTertiaryColor1]];
-    }
-    [stepViewController.step setTitle:@"Good Job!"];
-  
-    if ([stepViewController.step.identifier isEqualToString:kInstructionStepIdentifier]) {
-        UILabel *label = ((UILabel *)((UIView *)((UIView *)((UIView *) ((UIScrollView *)stepViewController.view.subviews[0]).subviews[0]).subviews[0]).subviews[0]).subviews[2]);
-        label.text = NSLocalizedStringWithDefaultValue(@"APH_MEMORY_STEP_INSTRUCTION", nil, APHLocaleBundle(),
```

Comment:
Does this label get set somewhere else? I'm not seeing it having been moved anywhere in your changes.